src/couch/src/couch_debug.erl

```diff
@@ -18,6 +18,18 @@
     opened_files_contains/1
 ]).
 
+-export([
+    process_name/1,
+    link_tree/1,
+    link_tree/2,
+    mapfold/3,
+    map/2,
+    fold/3,
+    linked_processes_info/2,
+    print_linked_processes/1,
+    ps/1
```

Comment:
Very nice. I like that idea.

Would it make sense to just add this as a special clause to print_linked_processes as opposed to have print_linked processes then also have ps as well?